Firebase/Core/FIRLogger.m

```diff
@@ -233,3 +233,16 @@ void FIRLogBasic(FIRLoggerLevel level,
 FIR_LOGGING_FUNCTION(Debug)
 
 #undef FIR_MAKE_LOGGER
+
+@implementation FIRLoggerWrapper
+
++(void) FIRLogBasicWrapper:(FIRLoggerLevel)level
```

Comment:
Naming suggestion, since FIRLoggerWrapper is the object itself:

+(void) logWithLevel:service:code:message:args:

?

That makes it easier to reason about: `[FIRLoggerWrapper logWithLevel:service:code:message:args:`

Thoughts?